### PR TITLE
add container-app-create quickstart

### PR DESCRIPTION
**Pull request by jbpaux**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* Add quickstart for new App Container service in preview

### CODE REVIEW

**Comment by jbpaux** [comment]
I guess it's failing because Microsoft.Web RP needs to be reregistered ? It's deploying fine on my sub.

File: quickstarts/microsoft.web/container-app-create/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Blank Template
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/container-app-create/README.md:1]: fix title

File: quickstarts/microsoft.web/container-app-create/main.bicep [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+@allowed([
+  'northcentralusstage'
+  'westcentralus'
+  'eastus'
+  'westeurope'
+  'jioindiawest'
+  'northeurope'
+  'canadacentral'
+])
+param location string = 'eastus' //cannot use resourceGroup().location since it's not available in most of regions
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/container-app-create/main.bicep:20]: remove defaultValue

**Review by bmoore-msft — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.web/container-app-create/main.bicep [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  'westcentralus'
+  'eastus'
+  'westeurope'
+  'jioindiawest'
+  'northeurope'
+  'canadacentral'
+])
+param location string //cannot use resourceGroup().location since it's not available in most of regions
+
+@description('Specifie the docker container image to deploy.')
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/container-app-create/main.bicep:22]: fix typo

File: quickstarts/microsoft.web/container-app-create/main.bicep [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+param containerImage string = 'mcr.microsoft.com/azuredocs/containerapps-helloworld:latest'
+
+@description('Specifies the container port.')
+param targetPort int = 80
+
+@description('Number of CPU cores the container can use. Can be with a maximum of two decimals.')
+param cpuCore string = '0.5'
+
+@description('Amount of memory (in gibibytes, Gi) allocated to the container up to 4Gi. Can be with a maximum of two decimals. Ratio with CPU cores must be equal to 2.')
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/container-app-create/main.bicep:31]: fix typo

File: quickstarts/microsoft.web/container-app-create/metadata.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "type": "QuickStart",
+  "itemDisplayName": "Creates a Container App within a Container App Environment.",
+  "description": "Create a Container App Environment with a basic Container App. It alsy deploys a Log Analytics Workspace to store logs.",
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/container-app-create/metadata.json:5]: fix typo

File: quickstarts/microsoft.web/container-app-create/metadata.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+  "summary": "Create a Container App Environment with a basic Container App. It alsy deploys a Log Analytics Workspace to store logs.",
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/container-app-create/metadata.json:6]: fix typo

File: quickstarts/microsoft.web/container-app-create/main.bicep [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+param memorySize string = '1'
+
+@description('Minimum number of replica that will be deployed')
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/container-app-create/main.bicep:34]: replica should be plural

File: quickstarts/microsoft.web/container-app-create/main.bicep [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+@minValue(0)
+param minReplica int = 1
+
+@description('Maximum number of replica that will be deployed')
```
Comment (bmoore-msft) [inline quickstarts/microsoft.web/container-app-create/main.bicep:38]: replica should be plural

**Review by bmoore-msft — CHANGES_REQUESTED** [review]
A few typos - also check the pipeline results for the best practice fixes that are needed

File: quickstarts/microsoft.web/container-app-create/main.bicep (hunk 8 above)
Comment (jbpaux) [reply quickstarts/microsoft.web/container-app-create/main.bicep]: done

File: quickstarts/microsoft.web/container-app-create/main.bicep (hunk 7 above)
Comment (jbpaux) [reply quickstarts/microsoft.web/container-app-create/main.bicep]: done

File: quickstarts/microsoft.web/container-app-create/metadata.json (hunk 6 above)
Comment (jbpaux) [reply quickstarts/microsoft.web/container-app-create/metadata.json]: done

File: quickstarts/microsoft.web/container-app-create/metadata.json (hunk 5 above)
Comment (jbpaux) [reply quickstarts/microsoft.web/container-app-create/metadata.json]: done

File: quickstarts/microsoft.web/container-app-create/main.bicep (hunk 4 above)
Comment (jbpaux) [reply quickstarts/microsoft.web/container-app-create/main.bicep]: not sure where the typo is ? It's gibibytes GiB.

File: quickstarts/microsoft.web/container-app-create/main.bicep (hunk 3 above)
Comment (jbpaux) [reply quickstarts/microsoft.web/container-app-create/main.bicep]: done

**Comment by jbpaux** [comment]
Regarding the BPA, there are warning in Bicep linter.
Reference here : https://github.com/Azure/bicep/issues/784 in the latest comments.
Hopefully it's good now. Sorry for the typos

**Comment by jbpaux** [comment]
Should be good now :)

**Pull request merged**